src/client/components/Modal/Modal.css

```diff
@@ -0,0 +1,44 @@
+* {
```

Comment:
Global styles do not belong in a component CSS. Use a separate PR for that and adjust it in the global CSS.